var/spack/repos/builtin/packages/py-mpld3/package.py

```diff
@@ -13,9 +13,12 @@ class PyMpld3(PythonPackage):
     homepage = "https://mpld3.github.io/"
     pypi = "mpld3/mpld3-0.3.tar.gz"
 
+    version('0.5.5', sha256='b080f3535238a71024c0158280ab4f6091717c45347c41c907012f8dd6da1bd5')
     version('0.3', sha256='4d455884a211bf99b37ecc760759435c7bb6a5955de47d8daf4967e301878ab7')
 
     depends_on('python@2.6:2.8,3.3:', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
     depends_on('py-matplotlib@1.3:', type=('build', 'run'))
+    depends_on('py-matplotlib@3.2.1:', type=('build', 'run'), when='@0.5.5:')
     depends_on('py-jinja2@2.7:', type=('build', 'run'))
+    depends_on('py-jinja2@2.11.2:', type=('build', 'run'), when='@0.5.5:')
```

Comment:
Where do these dependency versions come from? README says something different and documentation still has the old numbers.

Reply:
From requirements.txt. Although I guess that README is a better source of information.

Reply:
I generally trust `setup.py`, `setup.cfg`, or `pyproject.toml` and ignore the README, requirements.txt, and documentation since the former are used by `pip` and the latter aren't. I really wish Python's build system was more uniform...